File: Fs.Units/Compound/Inductance.fs
namespace Fs.Units
    
[<AutoOpen>]
module Inductance =
    
    /// Microhenry
    [<Measure>] type uH

    /// Millihenry
    [<Measure>] type mH

    /// Henry
    [<Measure>] type H

    [<RequireQualifiedAccess>]
    module MicroHenry =
        
        let create (x: float) = x * 1.0<uH>
        let toMilliHenrys (x: float<uH>) = x * 1.0<mH> / 1e+3<uH>
        let toHenrys (x: float<uH>) = x * 1.0<H> / 1e+6<uH>
        
    [<RequireQualifiedAccess>]
    module MilliHenry =
        
        let create (x: float) = x * 1.0<mH>
        let toMicroHenrys (x: float<mH>) = x * 1.0<uH> / 1e-3<mH>
        let toHenrys (x: float<mH>) = x * 1.0<H> / 1e+3<mH>
        
    [<RequireQualifiedAccess>]
    module Henry =
        
        let create (x: float) = x * 1.0<H>
        let toMicroHenrys (x: float<H>) = x * 1.0<uH> / 1e-6<H>
        let toMilliHenrys (x: float<H>) = x * 1.0<mH> / 1e-3<H>
    

File: Fs.Units.CSharp/Compound/Inductance.cs
// ReSharper disable once CheckNamespace
namespace Fs.Units.CSharp.Inductance;

public static class MicroHenryExtensions
{
    public static double MicroHenryToMilliHenrys(this double x) => MicroHenry.toMilliHenrys(x);
    public static double MicroHenryToHenrys(this double x) => MicroHenry.toHenrys(x);
}

public static class MilliHenryExtensions
{
    public static double MilliHenryToMicroHenrys(this double x) => MilliHenry.toMicroHenrys(x);
    public static double MilliHenryToHenrys(this double x) => MilliHenry.toHenrys(x);
}

public static class HenryExtensions
{
    public static double HenryToMicroHenrys(this double x) => Henry.toMicroHenrys(x);
    public static double HenryToMilliHenrys(this double x) => Henry.toMilliHenrys(x);
}